setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name=DisplayCAL
 version=attr: DisplayCAL.VERSION
-license_file = LICENSE.txt
+license_files = LICENSE.txt
```

Comment:
`license_file` was renamed to `license_files`. The warning about it was thrown multiple times during the build.

Reply:
true, pip is also complaining about it